integrations/quick-start-guide.md

```diff
@@ -0,0 +1,48 @@
+# Integration Management Quickstart Guide
+In this quickstart guide, we will go over key features for integration management.
+
+## Integrations Page
+The integrations page is your home for adding and viewing integrations connected to your Aqueduct account. You can naviate to integrations page by clicking `Integrations` in the left menu bar. This page has two main sections: [add an integration](#add-an-integration) and [connected integrations](#connected-integrations).
```

Comment:
Nit -- I wouldn't say "account" because it's not the enterprise version. I'd just say "to Aqueduct".

Reply:
typo: clicking *on* `Integrations